Volha_Ivanova/Module_4_Framework/Design_Patterns/my-app/src/main/java/com/epam/cdp/volha_ivanova/task1/loggerDecorator/LoggerDecorator.java

```diff
@@ -0,0 +1,13 @@
+package com.epam.cdp.volha_ivanova.task1.loggerDecorator;
+
+public class LoggerDecorator implements Logger {
```

Comment:
I suppose the class can be combine with HTMLLogger. It should be HTMLLoggerDecorator class only.